### ISSUE TITLE
Error: Undefined '_react3.default.PropType.func'

### ISSUE THREAD

**Issue by naytun** [FaridSafi/react-native-gifted-chat#718]
#### Issue Description

React native Undefined is not an object(evaluating'_react3.default.PropType.func')

#### Solution
1) Check your version of react, If the version is > 16 then propTypes is inside a new package called prop-types
see : https://www.npmjs.com/package/prop-types

2) Then,
Add: import PropTypes from 'prop-types'
Change: React.PropTypes.... => PropTypes....

**Comment by xcarpentier** [comment]
Hello @naytun,
This is not related to this package or I miss something?

**Comment by naytun** [comment]
Hi,

First of all I like to mentioned that what you’ve done with Gifted Chat package is amazing and saves me a whole lot of development time!!
Yes it is not directly related to the package, but when I tried to use ‘Slack example’ it is giving me that error and I have to change the way propTypes declared. I didn’t know how to add this solution so I added it in issues.

Regards,

Nay Tun Thein
iNTEGRA Systems Co., Ltd.
